core/src/translate/data_type.rs

```diff
@@ -14,6 +14,17 @@ pub fn translate_data_type(sql_data_type: &SqlDataType) -> Result<DataType> {
             Ok(DataType::Int)
         }
         SqlDataType::Int8(None) => Ok(DataType::Int8),
+        SqlDataType::Int16 => Ok(DataType::Int16),
+        SqlDataType::Int32 => Ok(DataType::Int32),
+        SqlDataType::Int128 => Ok(DataType::Int128),
+        SqlDataType::UInt8 => Ok(DataType::Uint8),
+        SqlDataType::UInt16 => Ok(DataType::Uint16),
+        SqlDataType::UInt32 => Ok(DataType::Uint32),
+        SqlDataType::UInt64 => Ok(DataType::Uint64),
+        SqlDataType::UInt128 => Ok(DataType::Uint128),
+
+        SqlDataType::Float32 => Ok(DataType::Float32),
+        SqlDataType::Float64 => Ok(DataType::Float),
```

Comment:
Nice, then we no longer need to keep `UINT8`, .. in `SqlDataType::Custom(_)` that are now unreachable codes.
we can remove those.

Reply:
<img width="877" alt="image" src="https://github.com/user-attachments/assets/80c7ddab-ce64-495d-8f10-bd18b53053f4">

Reply:
If I remove those, these tests fail:

```
test!("INT8" => Ok(DataType::Int8));
test!("INT16" => Ok(DataType::Int16));
test!("INT32" => Ok(DataType::Int32));
test!("INT128" => Ok(DataType::Int128));

test!("UINT8" => Ok(DataType::Uint8));
test!("UINT16" => Ok(DataType::Uint16));
test!("UINT32" => Ok(DataType::Uint32));
test!("UINT64" => Ok(DataType::Uint64));
test!("UINT128" => Ok(DataType::Uint128));
```

Should I remove these tests?